crates/ibc/src/core/ics04_channel/handler/send_packet.rs

```diff
@@ -20,14 +20,14 @@ pub struct SendPacketResult {
 }
 
 pub fn send_packet(
```

Comment:
Can we add docstrings on top of all the functions, reminding the reader of our convention? Similar to [this](https://github.com/cosmos/ibc-rs/blob/e5ad6956b1c31b3f0b3ccf31c0f63d60ace5bd48/crates/ibc/src/core/ics04_channel/handler/chan_open_try.rs#L14).

Reply:
[e061956](https://github.com/cosmos/ibc-rs/pull/323/commits/e061956c56631c7e66f56b9a5673242a2e191e3a)